cfme/infrastructure/provider/rhevm.py

```diff
@@ -73,13 +80,14 @@ def __init__(self, name=None, endpoints=None, zone=None, key=None, hostname=None
         self.end_ip = end_ip
         if ip_address:
             self.ip_address = ip_address
+        if self.appliance.version < self.tls_since_version:
+            self.endpoints['default'].delete_tls_info()
 
     @property
     def view_value_mapping(self):
         return {
             'name': self.name,
-            'prov_type': version.pick({version.LOWEST: 'Red Hat Virtualization Manager',
```

Comment:
**QUESTION:** Are we no longer supporting versions that said 'Red Hat Virtualization Manager'?

Reply:
I was recently testing 5.7.4.3 and there is 'Red Hat Virtualization' so I suppose it was backported. Because 5.7 is as old as it gets.

Reply:
Note the framework is only being written to support 58z and 59z  (and of course MIQ) at this time, so if 58z+ has `Red Hat Virtualization` then its fine to remove the version_pick.

In this case it looks like the change was made in a specific version of CFME - 5.8.0.10, and thus 5.8.0.9 would still use `Red Hat Virtualization Manager`.

I think this version pick should be left as-is for now, as its not required for what you're trying to resolve here with TLS configuration.


Note in general this fix, in the master branch, is really relevant for testing CFME 5.8.0.0+, and thus this fix will only have an effect when running against these older versions _still greater than 5.8.0.0_.  If your intent is to test 5.7z, and this fix is being added to support that, its slightly misguided and will need to be backported to the `5.8-breakpoint-patched` branch, which is run when testing 57z.

Since the version you trigger off of is in the 58z stream, the fix is still relevant here - it just won't be used (from master branch) when testing 57z.